include/mxnet/ndarray.h

```diff
@@ -1071,7 +1227,11 @@ class NDArray {
         // free storage
         Storage::Get()->Free(aux_handles[i]);
         // init aux storage
-        aux_handles[i] = Storage::Get()->Alloc(aux_bytes, ctx);
+        aux_handles[i] = Storage::Get()->Alloc(aux_bytes, ctx
+#if MXNET_ENABLE_STORAGE_TAGGING
+          , shandle.tag + ":" + AuxIdxToString(i)
+#endif  // MXNET_ENABLE_STORAGE_TAGGING
```

Comment:
can we avoid this preprocessor directive between a statement. To elaborate, change this to:
```
# if MXNET_ENABLE_STORAGE_TAGGING
aux_handles[i] = Storage::Get()->Alloc(aux_bytes, ctx, shandle.tag + ":" + AuxIdxToString(i));
#else 
aux_handles[i] = Storage::Get()->Alloc(aux_bytes, ctx);
#endif

```

Reply:
I previously also tried to avoid adding preprocessor directive to between a statement to avoid adding the `NOLINT(*)`statement. However, for functions that have a long list of arguments, this can be a nightmare. It also makes it harder to maintain as you always need to modify both code segments.